src/IniFileParser/Parser/IniDataParser.cs

```diff
@@ -350,7 +350,7 @@ protected virtual void ProcessKeyValuePair(string line, IniData currentIniData)
             // get key and value data
             string key = ExtractKey(line);
 
-			if (string.IsNullOrEmpty(key) && Configuration.SkipInvalidLines) return;
+            if (string.IsNullOrEmpty(key) && Configuration.SkipInvalidLines) return;
```

Comment:
Then I also remembered it was only introduced in .Net 4.0, so... yeah, forget the suggestion altogether.